clang/lib/Serialization/ASTReader.cpp

```diff
@@ -7477,6 +7477,10 @@ void TypeLocReader::VisitBTFTagAttributedTypeLoc(BTFTagAttributedTypeLoc TL) {
   // Nothing to do.
 }
 
+void TypeLocReader::VisitOverflowBehaviorTypeLoc(OverflowBehaviorTypeLoc TL) {
+  // Nothing to do.
+}
```

Comment:
Are you sure there is nothing to do? Don't you have to serialize the attribute location?

Reply:
Right. Done. thanks for spotting this.

Does this just help with pre-compiled headers?